src/utils/markets.ts

```diff
@@ -1,5 +1,25 @@
 import { Meet, Race } from "../types/meets";
 
+// MarketId 11 chars
+// AAAAAABBBCC
+// A = date as days since epoch
+// B = location code
+// C = race number
+export const makeMarketId = (
```

Comment:
unit test maybe? its not a formatting function though so not sure how valuable the test would be